### DISCUSSION TITLE
Would extended screen (Ipad pro, ipad 16.2) works in VM

### DISCUSSION THREAD

**Discussion by XDinEuro** [utmapp/UTM#5047] [UTM Help (iOS)]
A short question:

Now Ipad 16.2 supports extended screen, see [https://www.youtube.com/watch?v=dWMjpN4YILw&t=303s](https://www.youtube.com/watch?v=dWMjpN4YILw&t=303s).

Would this work also when I open VM display ?

If it works, should I config display in VM or in ipad itself ?

**Comment by osy** [comment]
To use extended display you need to set up at least two display devices in the settings. (Or one if you want to mirror it)

**Reply by XDinEuro** [reply]
> To use extended display you need to set up at least two display devices in the settings. (Or one if you want to mirror it)

In ipad setting or setting inside linux VM ?
